packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.pcss

```
@@ -12,6 +12,9 @@
 
 [part='submit-button'] {
   @apply w-10;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

There's a tailwind class for all of these

```suggestion
  @apply w-10 flex justify-center items-center;
--


<br class="Apple-interchange-newline">;
```

---

```suggestion
  @apply w-10 flex justify-center items-center;
```